test/kernel/integration/LiveUpdate/CMakeLists.txt

```diff
@@ -24,7 +24,7 @@ set(SOURCES
 # DRIVERS / PLUGINS:
 set(DRIVERS
     virtionet
-    boot_logger
+    #boot_logger
```

Comment:
Remove entirely or will it come back later ? TODO ?

Reply:
It is probably meant to be always-on, as if it crashes it should print as much as possible.